src/overlays/actors/ovl_Obj_Iceblock/z_obj_iceblock.h

```diff
@@ -27,8 +27,8 @@ typedef struct {
 } ObjIceBlockUnkStruct2; // size = 0x10
 
 typedef struct {
-    f32 unk_00;
-    f32 unk_04;
+    /* 0x0 */ f32 unk_00;
+    /* 0x4 */ f32 unk_04;
 } ObjIceBlockUnkStruct3; // size = 0x8
```

Comment:
And here (Vec2f?)

Reply:
Not quite sure, probably best to wait for documentation.